ResearchKit.podspec

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |s|
+  s.name         = 'ResearchKit'
+  s.version      = '1.0.0'
```

Comment:
Yup! A maintainer would create a release (`1.0.0` matches the project's build version), and `pod trunk push` after merging :octocat: